README.md

```diff
@@ -1,44 +1,47 @@
-This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+# Moltin &mdash; React Demo Store
 
-You can find a guide to some of the most common tasks [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
- 
----
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app). You can find a guide to some of the most common tasks [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
 
-## Deploying the app with Heroku
+## Development
 
-[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
+```bash
+git clone https://github.com/moltin/react-demo-store.git
+cd react-demo-store
+yarn #or npm install
+yarn start # or npm start
+```
+
+Note: You will want to change the `client_id` inside `src/moltin.js` with your own moltin store credentials.
+
+This demo store uses the Redux "[ducks](https://github.com/erikras/ducks-modular-redux)" approach to bundling reducers and actions.
+
+## Deployment
+
+### Heroku
 
----
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
```

Comment:
Does this link pass credentials into the application? Or does the app already use some base credentials?

Reply:
@outrunthewolf this button uses the `app.json` which requests a new Heroku app to be deployed, which will ask for the `client_id` as an `ENV` variable.